### ISSUE TITLE
Make menu compatible with RTL sites

### ISSUE THREAD

**Issue by tlartaud** [marvinlabs/customer-area#327]
Move `.just-hide` CSS to left on RTL sites and make sure JS is working for the menu.

**Comment by IgorIlgiyaev** [comment]
Made some workaround for RTL

.cuar-menu-container .nav-container .menu-plus {
    left: 0 !important;
    right: auto !important;
}
.cuar-menu-container .nav-container .just-hide{
    opacity:1 !important;
}

**Comment by tlartaud** [comment]
Hi,

Thanks for the tip :)
However, be careful when using `!important` rules.

I suppose that WP add a `rtl` class to the body or html node that should be instead.

`
body.rtl .cuar-css-wrapper .cuar-menu-container .nav-container .menu-plus {
left: 0;
right: auto;
}
`

If you're used to work with Git, do not hesitate to submit a Pull Request if you want to participate in the development.

Regards.

**Comment by IgorIlgiyaev** [comment]
Yeah I know about the important rule and I used it because without it it
didn't worked.
The body class I just copied what I saw in the inspect and used it :-)

Will submit stuff to Git if needed :-)
